### PR TITLE
Update site security page to match current flutter security policy

### PR DESCRIPTION
**Pull request by rmRizki**
_Description of what this PR is changing or adding, and why:_
Match [security page](https://flutter.dev/security) with current [flutter security policy](https://github.com/flutter/flutter/security/policy).

also refer to this [PR](https://github.com/flutter/.github/pull/30)

_Issues fixed by this PR (if any):_

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/security/index.md [hunk 1]
```diff
@@ -80,7 +80,19 @@ Any vulnerability reported for flutter websites like
...
 
-Flutter doesn't have a bug bounty program.
+## Bug Bounty programs
+
+Contributing teams are welcome to include Flutter within the scope
+of their bug bounty programs. To have your program listed, please
+contact `security@flutter.dev`.
+
+Google considers Flutter to be in scope for the [Google Open Source
+Software Vulnerability Reward Program][]. For expediency, we ask that
```
Comment (domesticmouse) [inline src/security/index.md:90]: This is being flagged as an invalid link.

File: src/security/index.md [hunk 2]
```diff
@@ -80,7 +80,19 @@ Any vulnerability reported for flutter websites like
...
+reporters please contact `security@flutter.dev` before using Google's
+vulnerability reporting flow.
+
+[Google Open Source Software Vulnerability Reward Program]:
```
Comment (domesticmouse) [inline src/security/index.md:94]: This looks wrong. It should be indented, and a single line, IIUC.

**Review by domesticmouse — COMMENTED** [review]
Invalid link

File: src/security/index.md (hunk 2 above)
Comment (rmRizki) [reply src/security/index.md]: updated. thanks for the review.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @atsansone

**Review by atsansone — APPROVED** [review]

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**